docs/source/usage/plugins/energyHistogram.rst

```diff
@@ -65,10 +65,26 @@ You can quickly plot the data in Python with:
    from picongpu.plugins.energy_histogram import EnergyHistogram
    import matplotlib.pyplot as plt
 
+   # load data
    energy_histogram = EnergyHistogram('/home/axel/runs/lwfa_001')
-   energies, bins = energy_histogram.get('e', iteration=2000)
+   counts, bins = energy_histogram.get('e', iteration=2000)
+
+   # unit conversion
+   MeV = 1.e-3  # keV to MeV
+
+   # plotting
+   plt.plot(bins * MeV, counts)
+
+   # range
+   ax = plt.gca()
+   # log scale example
+   # ax.set_yscale('log')
+   # ax.set_ylim([1.e7, 1.e12])
```

Comment:
QUESTION:
@ax3l Do you want to keep this dead code?

Reply:
yes, people always look this part up but it is not very simple to auto-express it when I tried via np.min/max, e.g. for single-bin distributions it will always look odd.